src/private.h

```diff
@@ -48,7 +54,7 @@ typedef struct _gpuarray_blas_ops gpuarray_blas_ops;
 struct _gpucontext {
   GPUCONTEXT_HEAD;
   void *ctx_ptr;
-  void *private[7];
+  void *private[8];
```

Comment:
Do you really need to grow this?  If you don't then it's best to leave it as it is.  There is a static assertion in the source that will trigger if it's too small.